tuto1/src/simple_declare.ml

```diff
@@ -9,7 +9,7 @@ let edeclare ident (_, poly, _ as k) ~opaque sigma udecl body tyopt imps hook =
      (Option.List.cons tyopt [body]) in
   let sigma = Evd.restrict_universe_context sigma uvars in
   let univs = Evd.check_univ_decl ~poly sigma udecl in
-  let ubinders = Evd.universe_binders sigma in
+  let ubinders = Evd.evar_universe_context sigma in
```

Comment:
If the function that is called is not called universe_binders anymore, I suggest the result should be stored in a local variable named ucontext instead of ubinders.

Reply:
Indeed. Renaming done. Thanks for the remark.